src/main/java/uk/gov/di/ipv/cri/common/library/service/SessionService.java

```diff
@@ -53,6 +53,8 @@ public UUID saveSession(SessionRequest sessionRequest) {
         sessionItem.setClientId(sessionRequest.getClientId());
         sessionItem.setRedirectUri(sessionRequest.getRedirectUri());
         sessionItem.setSubject(sessionRequest.getSubject());
+        sessionItem.setPersistentSessionId(sessionRequest.getPersistentSessionId());
```

Comment:
Update `SessionServiceTest.java` to cover these amends

Reply:
Added to shouldCallCreateOnSessionDataStore.